services/apps/search_sync_worker/src/repo/member.repo.ts

```diff
@@ -159,6 +159,7 @@ export class MemberRepository extends RepositoryBase<MemberRepository> {
               m.score,
               m."lastEnriched",
               m."joinedAt",
+              m."manuallyCreated",
```

Comment:
Also please check `getTenantMembersForSync` and `checkMembersExists` methods in this file because they both also assume that a member is only a member if it has activities...

Reply:
Thanks for pointing it out! I've updated the methods now 😄